enterprise/templates/enterprise/enterprise_course_enrollment_page.html

```diff
@@ -41,36 +41,38 @@ <h2 class="course-confirmation-title">{{ confirmation_text }}</h2>
           {{ course_short_description }}
           {{ view_course_details_text|link_to_modal:0 }}
         </div>
-        <form method="POST">
-          <input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}" />
-          {% if course_modes|length > 1 %}<div class="caption">{{ select_mode_text }}</div>{% endif %}
-          {% for course_mode in course_modes %}
-          <div class="radio-block">
-            <div class=" {% if course_modes|length > 1 %}radio{% endif %}">
-              {% if course_modes|length > 1 %}
-                <input type="radio" name="course_mode" id="radio{{ forloop.counter0 }}"{% if forloop.first %} checked="checked"{% endif %} value="{{ course_mode.mode }}" />
-              {% else %}
-                <input type="hidden" name="course_mode" id="radio{{ forloop.counter0 }}" value="{{ course_mode.mode }}" />
-              {% endif %}
-            </div>
-
-            <label for="radio{{ forloop.counter0 }}">
-              <strong class="title">{{ course_mode.title }}</strong>
-              <span class="price">
-                {{ price_text }}:
-                {% if course_mode.final_price and course_mode.original_price != course_mode.final_price %}
-                  <strike>{{ course_mode.original_price }}</strike> {{ course_mode.final_price }}
-                  <div>{{discount_text|safe }}</div>
+        {% if course_enrollable %}
```

Comment:
This `if` is the only actual addition. The rest was just spacing for this new branch.